README.md

```diff
@@ -70,6 +72,12 @@ Offical Website: [tdengine.com](https://tdengine.com)
 - [TDengine Rust bindings](https://github.com/songtianyi/tdengine-rust-bindings) - It's a Rust bindings project for TDengine.
 - [TDenginex](https://github.com/lizhaochao/TDenginex) - TDengine Connector for Elixir.
 - [DBUtility.TDengine](https://github.com/cockroach888/GSA.MOLLE.ToolKits/tree/main/src/ToolKits.DBUtility) - C# Connector for TDengine. (.NET 6+)
+- [IoTSharp.Data.Taos](https://www.nuget.org/packages/IoTSharp.Data.Taos/)  - TDengine's ADO.Net provider. (.Net Framework 4.6+, .Net Standard2.0+,.Net 5.0+)  [more](https://github.com/IoTSharp/EntityFrameworkCore.Taos) 
+
+
+## Health Checks
```

Comment:
I don't think `Health Checks` can be an independent category now. You can place it in `Others` section till there are more content under `Health Checks`.